src/crimson/osd/ops_executer.cc

```diff
@@ -378,12 +378,19 @@ OpsExecuter::watch_errorator::future<> OpsExecuter::do_op_notify_ack(
       }
       return seastar::do_for_each(obc->watchers,
         [ctx=std::move(ctx)] (auto& kv) {
-          const auto& [key, watchp] = kv;
+          auto [key, watchp] = kv;
```

Comment:
can we keep the original version. i think it helps to avoid the overhead of copy constructions. and `const` helps with the readability.